run.sh

```diff
@@ -116,8 +119,12 @@ main() {
 
   h3 "Adjusting file permissions"
   groupadd -f docker && usermod -aG docker www-data
-  find /app -type d -exec chmod 755 {} \;
-  find /app -type f -exec chmod 644 {} \;
+  if [[ "${PHP_ENV}" =~ ^dev ]]; then
+    chmod -R 777 /app
```

Comment:
Changing permissions to 777 in any situation is generally always a bad idea. The main issue is ownership, not permissions.

So the real "fix" for this would be to just create a user other than root and bootstrap with that user.

The fix would have to also be followed with a few fixes of the `run.sh` file because right now it's tailored around being root.